app/views/trips/index.erb

```diff
@@ -1,7 +1,7 @@
 <h1>All Trips</h1>
 
 <div>
-  <% @trips[1..30].each do |trip| %>
+  <% @trips[0..30].each do |trip| %>
```

Comment:
nice good catch! I'm playing around with pagination now so hopefully I can get this done tonight and we won't have to have that stub.